### ISSUE TITLE
how do you choose your cuhk03 dataset  ?

### ISSUE THREAD

**Issue by Phoebe-star** [zlmzju/part_reid#3]
how  about  your   gallery  and  probe  ?
and  training data

**Comment by zlmzju** [comment]
The official CUHK03 dataset provides the splits for training and test. I follow the paper of CUHK03 to use their protocol (choosing the gallery and probe for each identity randomly).

**Comment by Phoebe-star** [comment]
the code can convert to tensorflow ?

  and  when  training ,  your triplet loss  is  always  small ?
like :  Step: 67, Learning rate: 0.009951, Train loss: 0.031987
         Step: 3300, Learning rate: 0.007951, Train loss: 0.022587
         Step: 5200, Learning rate: 0.006951, Train loss: 0.030987

**Comment by Phoebe-star** [comment]
how do you do resize images?
you directly resize 224x224 ?  not do crop? or others?
and how about your Data Augmentation

**Comment by jianwu585218** [comment]
I'm trying to convert to tensorflow,but get some troubles.
Did you successfully convert to tensorflow ?

**Comment by Phoebe-star** [comment]
how about your troubles?

**Comment by jianwu585218** [comment]
In the Part Map Detector :
![image](https://user-images.githubusercontent.com/34296200/38592773-85a4f61a-3d70-11e8-8e56-7a3effb0f053.png)

I think the mk(x,y) is a 1*1 local convolutional layer which not shared weight , isn't it ?
but i don't know how to make it .
I need your help, thanks very much.

**Comment by jianwu585218** [comment]
Hi,when i'm training my model by tensorflow, I found the triplet loss is drop hard.I want to know have you success reappearance ? Thank you very much~

**Comment by Phoebe-star** [comment]
yes, you are right ,  mk(x,y) is a 1*1 local convolutional layer which not shared weight
have you been used hard example? if you used, the training will be hard.

**Comment by jianwu585218** [comment]
I have not use the hard example ,I used the random triplet ,and the loss can't be  decline

**Comment by Phoebe-star** [comment]
Do you have use the  googlenet pretrain from imagenet?   how about your loss?

**Comment by jianwu585218** [comment]
Thanks for your reply! 
I use the vgg16 pretrain model from imagenet , my loss is original triplet loss ,the triplet pair is random generated , such as :


Do you use the hard triplet ?

**Comment by jianwu585218** [comment]
Thanks for your reply!
I use the vgg16 pretrain model from imagenet , my loss is original triplet loss ,the triplet pair is random generated , such as :
![image](https://user-images.githubusercontent.com/34296200/39115007-224f2e12-4713-11e8-9d3d-f4df79d4460a.png)

Do you use the hard triplet ?

**Comment by Phoebe-star** [comment]
I think your vgg have some problem? like the whights don't change . why don't you use the googlenet?
if use original triplet loss ,the loss will be easy to train ,
but the hard triplet , I guess the loss will be difficult to train

**Comment by jianwu585218** [comment]
Thanks for your reply again~
Because i  am used to use the vgg , i have not used googlenet  before , and i don't know how to use the local conv layer by tensorflow .
I'm a new hand of tensorflow ,  I have tried this project for nearly a month.
And can you send your demo to learning ?  Although I know this is an impolite request , you can refuse me ~
Thank you very much~

**Comment by Phoebe-star** [comment]
I also use someone  github , but  can I  point it here?
after all, here is zlmzju 's  github

**Comment by jianwu585218** [comment]
Can you send the github to my email?
This is my email :1449214372@qq.com.
Thank you very much ,you and zlm is my star😊
